CMakeLists.txt

```diff
@@ -468,9 +468,29 @@ if (BUILD_TOOLS)
   )
 
   # wasm2c
+  set(TEMPLATE_PY ${WABT_SOURCE_DIR}/scripts/gen-wasm2c-templates.cmake)
```

Comment:
Maybe call this something else now that its not python anymore?

Reply:
Nice catch, just renamed to TEMPLATE_CMAKE